app/models/reference/CUSCode.scala

```diff
@@ -17,15 +17,31 @@
 package models.reference
 
 import cats.Order
+import config.FrontendAppConfig
 import models.Selectable
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{__, Json, OFormat, Reads}
 
 case class CUSCode(code: String) extends Selectable {
 
+  override def toString: String = code
+
   override val value: String = code
 }
 
 object CUSCode {
+
+  def reads(config: FrontendAppConfig): Reads[CUSCode] =
+    if (config.isPhase6Enabled) {
+      (__ \ "key").read[String].map(CUSCode.apply)
```

Comment:
I suspect this should be "value". Let me double check with Dave Gregory. I'll update customs-reference-data test only resources accordingly.

Reply:
On reflection, key is probably correct. I have informed that we may have `value: null` or similar, but it is up to them how they choose to handle that. Haven't heard back from him but happy to mark this as resolved 🙂